apps/desktop/src/renderer/atoms/workflow-board.ts

```
@@ -0,0 +1,99 @@
+
+        setBoard(response.snapshot)
+        setError(response.snapshot.lastError?.message ?? null)
+      } catch (error) {
+        if (cancelled) {
+          return
+        }
+
+        setError(error instanceof Error ? error.message : String(error))
+      } finally {
+        if (cancelled) {
+          return
```

Biome (2.4.10)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)